examples/js/package.json

```diff
@@ -16,11 +16,11 @@
     "@lingui/macro": "^3.5.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.3",
-    "@babel/preset-env": "^7.12.1",
+    "@babel/core": "^7.20.12",
+    "@babel/preset-env": "^7.20.2",
```

Comment:
Babel update was a must to support ES2022 syntax in babel parser

Due to monorepo babel should be updated everywhere  (it brings troubles if different packages has babel versions mismatch)